src/virtual-select.js

```diff
@@ -696,6 +697,10 @@ export class VirtualSelect {
     this.setSearchValue(e.target.value, true);
   }
 
+  toPreventEvent(e) {
```

Comment:
Add a brief comment explaining its purpose and why it stops propagation of the change event, since, by looking at the code it's not easy to understand.